global.d.ts

```diff
@@ -0,0 +1,12 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      DB_URL: string;
```

Comment:
We should not add these fields, since we cannot be sure the the process is always ran with these environment variable, so it may breaks type safety.

Reply:
ok, I'll remove the rest and just include `DB_URL`